README.adoc

```diff
@@ -11,14 +11,14 @@
 :page-duration: 15 minutes
 :page-releasedate: 2018-10-19
 :page-description: Learn how to secure a web application through authentication and authorization.
-:page-tags: ['Java EE', 'Jakarta EE', 'Security']
+:page-tags: ['Jakarta EE', 'Security']
 :page-related-guides: ['microprofile-jwt', 'cdi-intro']
 :page-guide-category: none
 :page-permalink: /guides/{projectid}
-:common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
+:common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/tags-twyb-intro
```

Comment:
We can't link to another branch other than `master`. It's confusing and not official.

Reply:
I needed to use the tags from that branch in this PR (https://github.com/OpenLiberty/guides-common/pull/376). So that's why that branch was used here, since it has not been merged to master yet. Once that guides-common branch is merged to master, then i will link it to master